### DISCUSSION TITLE
Filament vs. Object flow ratio

### DISCUSSION THREAD

**Discussion by cutmoney** [OrcaSlicer/OrcaSlicer#1627] [General]
Does anyone know whether the flow ratio value in the filament properties combines with or replace/override the flow ratio for the plate/object? In other words, if I set the filament flow ratio to .95, then set the flow ratio for the plate/object in the Advanced tab to .95 also, does that result in an overall flow ratio of .9025 (.95 * .95) or just .95?

**Comment by mdrichardson** [comment]
I'd like to know this, too. I'm doing the calibration and after pass 1, I update the filament's flow ratio (say to 0.95 for example). But when I do pass 2, the Object flow shows `[1, 0.99, 0.98...]` instead of `[0.95, 0.89, 0.93...]`

**Comment by mdrichardson** [comment]
aha! @cutmoney, found here: https://github.com/SoftFever/OrcaSlicer/discussions/1120

Object shows the modification of what's set in the filament
